.buildkite/publish/dra/init_dra_publishing.sh

```diff
@@ -142,12 +140,8 @@ if [[ "${PUBLISH_STAGING:-}" == "true" ]]; then
   setDraVaultCredentials
   export WORKFLOW="staging"
```

Comment:
Just realized this - do you want to publish both on `staging` and `snapshot` for the `main` branch? IIRC - `staging` should only be for numbered branches...

Reply:
You're correct, and I believe that's handled correctly here? https://github.com/elastic/connectors/pull/2749/files#diff-4ec243366bbf09e6e1da80e47528b1fab33c1245a8e9c993c5d7a65e5464f927R44-R46

Reply:
Correct - and I didn't see you already have the check for `PUBLISH_STAGING` above the block... 👍  all good